modules/ROOT/pages/afm-create-private-location.adoc

```diff
@@ -7,29 +7,28 @@ endif::[]
  
 You can create a private location as part of three tasks:
 
-* When you create a monitor
-* When you add a schedule to a monitor
-* When you modify a schedule in a monitor
+* When you xref:afm-create-monitor.adoc[create a monitor]
+* When you xref:afm-add-schedule.adoc[add a schedule] to a monitor
+* When you xref:afm-modify-schedule.adoc[modify a schedule] in a monitor
 
-You can also create a private location independently of these three tasks. Then, when you create a monitor, add a schedule, or modify a schedule, you can simply select the private location. The following procedure explains the steps for this last method.
+When you create a monitor, add a schedule, or modify a schedule, you can select the private location. This procedure explains the steps for this last method.
 
-For information about what a private location is and why you might want to create one, see xref:afm-monitoring-private-apis.adoc[Monitoring the Endpoints of Private APIs].
+For information about private locations and their uses, see xref:afm-monitoring-private-apis.adoc[Monitoring the Endpoints of Private APIs].
 
 == Before You Begin
 
-Ensure that your organization in Anypoint Platform has a subscription to Anypoint Virtual Private Cloud.
+Ensure that your Anypoint Platform organization has a subscription to Anypoint Virtual Private Cloud.
```

Comment:
Anypoint Platform Private Cloud Edition is the only product name that I can find that comes close to this.
There is no "Anypoint Virtual Private Cloud" as a named entity, as far as I can tell.

Reply:
I think this is referring to Anypoint Virtual Private Cloud (VPC):
https://docs.mulesoft.com/runtime-manager/virtual-private-cloud